tests/generic_relations_regress/tests.py

```diff
@@ -15,7 +14,6 @@
     Contact,
     Content,
     D,
-    Developer,
```

Comment:
You can rebase from the `main` branch and remove `Developer` model, that is now unused (see also 1ed8ca43f61138b8f8d6f92106c27060753ed4e7)

Reply:
Done :)